proto/regen/ecocredit/v1/query.proto

```diff
@@ -644,3 +696,61 @@ message BatchBalanceInfo {
   // cancelled, updated with a reduced quantity, or processed.
   string escrowed_amount = 5;
 }
+
+// ClassCreatorInfo is the human-readable class creator information.
+message ClassCreatorInfo {
+  // address is the address that is allowed to create credit classes
+  string address = 1;
+}
+
+// QueryAllowedClassCreatorsRequest is the Query/AllowedClassCreators request
+// type.
+//
+// Since Revision 1
+message QueryAllowedClassCreatorsRequest {
+  // pagination defines an optional pagination for the request.
+  cosmos.base.query.v1beta1.PageRequest pagination = 1;
+}
```

Comment:
Would be nice to maintain the order of messages we have with all the query request and response messages and then all the info structs rather than mixing the two.